viewer-prototype/src/browser/trace-viewer/components/utils/filtrer-tree/checkbox-component.tsx

```diff
@@ -27,7 +27,7 @@ export class CheckboxComponent extends React.Component<CheckboxProps> {
     };
 
     render(): JSX.Element {
-        return <div onClick={() => this.props.onChecked(this.props.id)} >
+        return <div onClick={() => this.props.onToggleCheck(this.props.id)} >
```

Comment:
Sorry, I missed that in my previous review. I have same comment than last review:
```
render(): JSX.Element {
        return <div onClick={this.onClickDiv} >
            <span style={{padding: 5}}>
                {this.renderCheckbox(this.props.checkedStatus)}
            </span>
            {this.props.name}
        </div>;
    }

    private onClickDiv = () => {
        this.props.onToggleCheck(this.props.id)
    }
```